pom.xml

```diff
@@ -57,80 +55,91 @@
-    <hpi.compatibleSinceVersion>1.5.0</hpi.compatibleSinceVersion>
-    <google.api.version>1.25.0</google.api.version>
-    <google.http.version>1.21.0</google.http.version>
-    <google.guava.version>30.0-jre</google.guava.version>
-    <google-oauth.version>1.0.0</google-oauth.version>
-    <storage.revision>158</storage.revision>
     <java.level>8</java.level>
-    <concurrency>5</concurrency>
+    <jenkins.version>2.319.1</jenkins.version>
+    <hpi.compatibleSinceVersion>1.5.0</hpi.compatibleSinceVersion>
+    <google-api.version>1.33.2</google-api.version>
+    <google-http.version>1.41.4</google-http.version>
+    <google-oauth.version>1.33.1</google-oauth.version>
+    <google-storage.revision>v1-rev20211201-1.32.1</google-storage.revision>
+    <guava.version>31.0-jre</guava.version>
     <skipIts>true</skipIts>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
-        <artifactId>bom-2.222.x</artifactId>
-        <version>21</version>
+        <artifactId>bom-2.319.x</artifactId>
+        <version>1148.v7261f385f859</version>
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      <dependency>
-        <groupId>com.google.errorprone</groupId>
-        <artifactId>error_prone_annotations</artifactId>
-        <version>2.6.0</version>
-      </dependency>
     </dependencies>
   </dependencyManagement>
   <dependencies>
     <dependency>
-      <groupId>commons-io</groupId>
-      <artifactId>commons-io</artifactId>
-      <version>2.8.0</version>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>credentials</artifactId>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>credentials</artifactId>
+      <artifactId>google-metadata-plugin</artifactId>
+      <version>0.3.1</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>structs</artifactId>
+      <artifactId>google-oauth-plugin</artifactId>
+      <version>1.0.6</version>
     </dependency>
     <dependency>
-      <groupId>org.jenkinsci.plugins</groupId>
-      <artifactId>pipeline-model-definition</artifactId>
-      <scope>test</scope>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>jackson2-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>ssh-credentials</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
+      <artifactId>structs</artifactId>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-step-api</artifactId>
     </dependency>
     <dependency>
-      <groupId>org.jenkins-ci.plugins</groupId>
-      <artifactId>ssh-credentials</artifactId>
+      <groupId>commons-io</groupId>
```

Comment:
🤔 does this not cause it to be bundled - it is not coming from core?

Reply:
Removed the dependency in https://github.com/jenkinsci/google-storage-plugin/pull/186/commits/942e23613636ec7a14565d2427ba4e8627060975